waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/MetaStoreProxyServer.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (C) 2015-2017 The Apache Software Foundation and Expedia Inc.
```

Comment:
I've added the missing comma(!) and updated the year.

Reply:
out of interest, will this ever be updated to Expedia Group?

Reply:
According to our lawyers, in the near future, no. I'm guessing it's the name of some legally registered entity which is hard/expensive to change.

Reply:
ahh okay